client/blocks/comment-detail/comment-detail-reply.jsx

```diff
@@ -2,18 +2,99 @@
  * External dependencies
  */
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import AutoDirection from 'components/auto-direction';
+import { getCurrentUser } from 'state/current-user/selectors';
 
 export class CommentDetailReply extends Component {
+	state = {
+		commentText: '',
+		haveFocus: false,
+	};
+
+	handleBlur = () => this.setState( { haveFocus: false } );
+
+	handleFocus = () => this.setState( { haveFocus: true } );
+
+	handleKeyDown = event => {
+		// Use Ctrl+Enter to submit comment
+		if ( event.keyCode === 13 && ( event.ctrlKey || event.metaKey ) ) {
+			event.preventDefault();
+			this.submit();
+		}
+	}
+
+	handleSubmit = event => {
+		event.preventDefault();
+		this.submit();
+	}
+
+	handleTextChange = event => this.setState( { commentText: event.target.value } );
+
+	submit = () => {
+		const comment = {
+			authorAvatarUrl: this.props.currentUser.avatar_URL,
+			authorName: this.props.currentUser.display_name,
+			authorUrl: this.props.currentUser.primary_blog_url,
+			parentId: this.props.commentId,
+			postTitle: this.props.postTitle,
+			content: this.state.commentText,
+			URL: this.props.postUrl,
+		};
+		this.props.submitComment( comment );
+		this.setState( { commentText: '' } );
+	}
+
 	render() {
 		const { translate } = this.props;
+		const { commentText, haveFocus } = this.state;
+
+		const hasCommentText = commentText.trim().length > 0;
+
+		const buttonClasses = classNames( 'comment-detail__reply-submit', {
+			'is-active': hasCommentText,
+			'is-visible': haveFocus || hasCommentText,
+		} );
+
+		const expandingAreaClasses = classNames( 'expanding-area', {
+			focused: haveFocus,
+		} );
 
 		return (
-			<div className="comment-detail__reply">
-				<textarea placeholder={ translate( 'Reply' ) }></textarea>
-			</div>
+			<form className="comment-detail__reply">
+				<div className={ expandingAreaClasses }>
+					<pre><span>{ commentText }</span><br /></pre>
```

Comment:
Yup.
I've tried removing it to no avail.

[Here](https://alistapart.com/article/expanding-text-areas-made-elegant) the explanation:
> The `pre` is our mirror. We need a `br` at the end of it to ensure that any trailing whitespace copied from the `textarea` is rendered by the browser correctly and not chewed up. The `span` element is therefore the one we actually update with the contents of the `textarea`.